src/components/drawing/index.js

```diff
@@ -56,7 +56,7 @@ drawing.setRect = function(s, x, y, w, h) {
  *
  * @return {boolean} :
  *  true if selection got translated
- *  false if selection got removed
+ *  false if selection could not get translated
  */
 drawing.translatePoint = function(d, sel, xa, ya) {
```

Comment:
To my surprise, `Drawing.translatePoint` is called only in `scatter/plot.js` and `box/plot.js`:

![image](https://cloud.githubusercontent.com/assets/6675409/26368787/b533534e-3fc0-11e7-9c08-74111af9879a.png)

Note that this patch does **not** affect the `box/plot.js` where points with non-numeric positions are filtered out of the data bind.